src/main/java/roomescape/member/MemberService.java

```diff
@@ -0,0 +1,46 @@
+package roomescape.member;
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import roomescape.auth.dto.LoginMember;
+import roomescape.exception.custom.reason.member.MemberEmailConflictException;
+import roomescape.exception.custom.reason.member.MemberNotFoundException;
+import roomescape.member.dto.MemberRequest;
+import roomescape.member.dto.MemberReservationResponse;
+import roomescape.member.dto.MemberResponse;
+
+@Service
+@AllArgsConstructor
+public class MemberService {
+
+    private final MemberRepository memberRepository;
+
+    public void createMember(final MemberRequest request) {
+        validateDuplicationEmail(request);
+
+        final Member notSavedMember = new Member(request.email(), request.password(), request.name(), MemberRole.MEMBER);
+        memberRepository.save(notSavedMember);
+    }
+
+    public List<MemberReservationResponse> readAllReservationsByMember(final LoginMember loginMember){
+        final Member member = memberRepository.findByEmail(loginMember.email())
+                .orElseThrow(() -> new MemberNotFoundException());
```

Comment:
생성자도 람다 대신 아래와 같이 사용할 수 있습니다.
```suggestion
                .orElseThrow(MemberNotFoundException::new);
```